ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -671,6 +690,44 @@ VAV_SD_DTM:
   - SD
   - DTM
 
+VAV_DSP_DFSS_DFVSC:
+  description: "Fan powered VAV with variable speed control and dual zone temperature setpoints."
+  is_canonical: true
+  implements:
+  - VAV
+  - DSP
+  - DFSS
+  - DFVSC
+
+VAV_DSP_HTZC_DFSS_DFVSC: 
+  description: "Fan powered VAV with variable speed control and electric heating controlling to dual zone temperature setpoints."
+  is_canonical: true
+  implements:
+  - VAV
+  - DSP
+  - HTZC
+  - DFSS
+  - DFVSC
+
+VAV_DFSS_DFVSC_DSP_HWZC:
+  description: "Fan powered VAV with variable speed control and hydronic heating controlling to dual zone temperature." 
+  is_canonical: true
+  implements:
+  - VAV
+  - DFSS
+  - DFVSC
+  - DSP
+  - HWZC
+
+VAV_DFSS_DFVSC_HWVM:
+  description: "Hydronic variable speed vav with heating water valve monitoring."
+  is_canonical: true
+  implements:
+  - VAV
+  - DFSS
+  - DFVSC
+  - HWVM
```

Comment:
Why is this monitoring and not control? How does the HW modulate and what does it control to?